### PR TITLE
Upgradle gradle

### PR DESCRIPTION
**Pull request by mar-kolya**

### CODE REVIEW

File: dd-trace-java.gradle [hunk 1]
```diff
@@ -49,10 +49,10 @@ buildScan {
...
 }
 
-ext.gradleWrapperVersion = '4.10.3'
-tasks.register("wrapper", Wrapper) {
-  gradleVersion = gradleWrapperVersion
-}
+ext.gradleWrapperVersion = '5.3.1'
+//tasks.register("wrapper", Wrapper) {
+//  gradleVersion = gradleWrapperVersion
+//}
```
Comment (tylerbenson) [inline dd-trace-java.gradle]: Can you keep this task?  I usually change the version then run the `wrapper` task to make sure all the scripts are upgraded properly.

File: dd-trace-java.gradle [hunk 2]
```diff
@@ -2,12 +2,12 @@ plugins {
   id 'io.franzbecker.gradle-lombok' version '1.14'
   id 'com.jfrog.artifactory' version '4.8.1'
   id 'com.jfrog.bintray' version '1.8.4'
-  id 'org.unbroken-dome.test-sets' version '1.5.2'
+    id 'org.unbroken-dome.test-sets' version '2.1.1'
```
Comment (tylerbenson) [inline dd-trace-java.gradle]: indentation...

File: dd-java-agent/instrumentation/aws-java-sdk-2.2/aws-java-sdk-2.2.gradle [hunk 3]
```diff
@@ -51,6 +51,7 @@ dependencies {
   testCompile project(':dd-java-agent:instrumentation:netty-4.1')
   // Needed by netty async instrumentation
   testCompile project(':dd-java-agent:instrumentation:java-concurrent')
+  testCompile group: 'software.amazon.awssdk', name: 'apache-client', version: '2.2.0'
```
Comment (tylerbenson) [inline dd-java-agent/instrumentation/aws-java-sdk-2.2/aws-java-sdk-2.2.gradle]: why was this needed?

File: dd-java-agent/instrumentation/hystrix-1.4/hystrix-1.4.gradle [hunk 4]
```diff
@@ -31,6 +31,7 @@ dependencies {
   testCompile project(':dd-java-agent:testing')
   testCompile project(':dd-java-agent:instrumentation:java-concurrent')
   testCompile project(':dd-java-agent:instrumentation:trace-annotation')
+  testCompile group: 'io.reactivex', name: 'rxjava', version: '1.0.7'
```
Comment (tylerbenson) [inline dd-java-agent/instrumentation/hystrix-1.4/hystrix-1.4.gradle]: why was this needed?

File: dd-trace-java.gradle (hunk 1 above)
Comment (mar-kolya) [reply dd-trace-java.gradle]: This task now exists in Gradle, we just need to configure it...

File: dd-java-agent/instrumentation/hystrix-1.4/hystrix-1.4.gradle (hunk 4 above)
Comment (mar-kolya) [reply dd-java-agent/instrumentation/hystrix-1.4/hystrix-1.4.gradle]: it yelled at me about not finding those classes

File: dd-java-agent/instrumentation/aws-java-sdk-2.2/aws-java-sdk-2.2.gradle (hunk 3 above)
Comment (mar-kolya) [reply dd-java-agent/instrumentation/aws-java-sdk-2.2/aws-java-sdk-2.2.gradle]: it yelled at me about not finding http client classes

File: dd-trace-java.gradle (hunk 2 above)
Comment (mar-kolya) [reply dd-trace-java.gradle]: fixed

File: dd-java-agent/instrumentation/aws-java-sdk-2.2/aws-java-sdk-2.2.gradle (hunk 3 above)
Comment (labbati) [reply dd-java-agent/instrumentation/aws-java-sdk-2.2/aws-java-sdk-2.2.gradle]: [Some work](https://gradle.org/whats-new/gradle-5/#fine-grained-transitive-dependency-management) was done on transitive dependencies in 5 vs 4 but looking at the changelog it doesn't seem to me that this is expected. Strange.

File: dd-trace-java.gradle (hunk 1 above)
Comment (tylerbenson) [reply dd-trace-java.gradle]: In that case, I suggest just delete it.

There used to be additional benefit with this task keeping other projects also in sync, but those are all part of the same project tree now.

Comment (tylerbenson) [reply dd-trace-java.gradle]: I guess you did...

Comment (mar-kolya) [reply dd-trace-java.gradle]: Yeah, we now have this:

```
wrapper {
  distributionType = Wrapper.DistributionType.ALL
  version = '5.3.1'
}
```

-- which seems to lock Gradle version and also downloads Gradle archive with sources to help debugging.
